### PR TITLE
Enforce consistent coding style

### PR DESCRIPTION
**Pull request by junwei0117**
Code consistency is important, so I suggest to follow the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript)

### CODE REVIEW

**Comment by ender503** [comment]
> Update example code
> 
> Let code consistent coding style

The commit subject should reflect your code changes, such as "Tweak coding styles" or "Add ESLint settings".

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "evm-lite-env",
   "version": "0.1.0",
-  "description": "Deploy environment for running Lightweight Ethereum node with interchangeable consensus.",
+  "description": "Deploy environment for running Lightweight EVM node with interchangeable consensus.",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (ender503) [inline package.json:7]: Add the scripts to lint the coding style with ESLint, such as
```javascript
  "lint": "eslint --fix <javascript folder>"
```
Add the ESLint package into your devDependency, we could not expect ESLint has been installed globally.

File: example/deploySmartContractLocally.js [hunk 2]
```diff
@@ -5,44 +5,45 @@ const solc = require('solc');
...
-    // Get keystore object from the keystore directory
-    // For the from address so we can decrypt and sign
-    const account = await dataDirectory.keystore.decrypt(from, password);
+  // Generate contract object with ABI and data
+  const contract = await evmlc.generateContractFromABI(ABI, data);
 
-    // Generate contract object with ABI and data
-    const contract = await evmlc.generateContractFromABI(ABI, data);
+  // Deploy and return contract with functions populated
+  const response = await contract.deploy(account, { parameters: [100000] });
```
Comment (ender503) [inline example/deploySmartContractLocally.js]: What is the `parameters` value used for? 🤔

**Comment by jserv** [comment]
Add a new file [STYLE-GUIDE.md](https://github.com/TangleID/TangleID/blob/master/STYLE-GUIDE.md) as TangleID does.

**Review by jserv — CHANGES_REQUESTED** [review]
Add style guide for further consistency.

File: package.json (hunk 1 above)
Comment (junwei0117) [reply package.json:7]: The ESLint package has already in the devDependency.

File: example/deploySmartContractLocally.js (hunk 2 above)
Comment (jserv) [reply example/deploySmartContractLocally.js]: The explanation is required. @mushroom0117, please clarify.

Comment (junwei0117) [reply example/deploySmartContractLocally.js]: It is meant to be the constructor parameters of the Contract.

Comment (jserv) [reply example/deploySmartContractLocally.js]: Insert some relevant comment?

Comment (junwei0117) [reply example/deploySmartContractLocally.js]: In the before example code, we use CrowdFunding contract for example.
so the [100000] is a list of the parameters, takes in a value for the limit to the total contribution.
But we have remove the hardcode, so the [100000] should be replace to an empty list.

Comment (jserv) [reply example/deploySmartContractLocally.js]: Roger. Refactor the sample code (maybe in another pull request).

**Pull request merged**